src/javascript/ast-value.ts

```diff
@@ -28,6 +28,8 @@ function literalToValue(literal: babel.Literal): LiteralValue {
   if (babel.isNullLiteral(literal)) {
     return null;
   }
+  // Any other literal value is treated as undefined.
+  return undefined;
```

Comment:
This is because `undefined` is what this function returns when it can't get a value, because it's from the lossy-goosy old days of the analyzer. If we wrote it today we'd return something with a more structured differentiation between intended results and failure

Reply:
👍